src/aws/auth.rs

```diff
@@ -676,4 +692,21 @@ mod tests {
             _ => panic!(),
         }
     }
+
+    #[test]
+    fn parsing_sign_false() {
+        let config = toml::from_str::<ComponentConfig>(
+            r#"
+            auth.sign = false
+        "#,
+        )
+        .unwrap();
+
+        match config.auth {
+            AwsAuthentication::Default { sign, .. } => {
+                assert!(!sign);
+            }
+            _ => panic!(),
+        }
```

Comment:
This could probably be more concise as a `assert!(matches!(...))`

Reply:
Mmm, true, though I like it as-is since it matches the other tests here.